app/views/downloads/download.html.erb

```diff
@@ -1,3 +1,7 @@
-<% if @error %>
-  The download link is either invalid, has expired, or the file has already been accessed. Please try exporting again. If you think you received this message in error, please contact the help desk.
-<% end %>
+<%= react_component('downloads/Downloads', {
+                      error: @download.nil?,
+                      download: @download,
+                      export_url: @export_url,
+                      time: (ADMIN_OPTIONS['download_destroy_wait_time'].to_i / 60),
+                      authenticity_token: form_authenticity_token
+                    }) %>
```

Comment:
wasn't this component and downloads controller removed?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/550/commits/ce66b4b53d8cdc6eaacaba087e6a5b1fdac91dcb